docs/families.docs.js

```diff
@@ -0,0 +1,232 @@
+const familyEndpoints = {
+  '/families': {
+    post: {
+      tags: ['families'],
+      summary: 'Created object families in response',
+      description: 'families router',
+      operationId: 'postLoggedFamily',
+      produces: ['application/json'],
+      security: {
```

Comment:
we don't need security option here or anywhere else, since we're passing directly `Authorization` hearer